packages/gitbook/src/components/Search/SearchContainer.tsx

```diff
@@ -23,13 +23,14 @@ interface SearchContainerProps {
     isMultiVariants: boolean;
     style: CustomizationSearchStyle;
     className?: string;
+    visibility?: 'desktop' | 'mobile';
```

Comment:
Should we choose a different name for this prop? Maybe `viewport`? Visibility I'd related with something that can be shown/hidden